_data/tools.yml

```diff
@@ -1034,6 +1034,24 @@
   description: Generate OpenAPI 3 specification for Laravel Applications.
   v2: false
   v3: true
+  
+- name: json-schema-spell-checker
+  category:
+    - miscellaneous
+  language: Node.js
+  github: https://github.com/mheap/json-schema-spell-checker
+  description: Check OpenAPI specification for spelling mistakes using mdspell
```

Comment:
```suggestion
  description: Check OpenAPI descriptions for spelling mistakes using mdspell.
```

Reply:
It can technically check anything. I use it to check titles and descriptions.

Reply:
Then maybe it should just be called `json-spell-checker` instead? Then another tool called openapi-spell-checker could be implementing json spell checker on known openapi keywords, like title, description, summary, etc for various components. What do you think?

Reply:
Oh I dont mean description fields, i mean API Descriptions. Check it out https://github.com/openapi-contrib/glossary